users/xulkal/process_records.c

```diff
@@ -1,56 +1,58 @@
 #include "process_records.h"
-
-#ifdef TAP_DANCE_ENABLE
-//Tap Dance Definitions
-qk_tap_dance_action_t tap_dance_actions[] = {
-  [COMM_QUOT]  = ACTION_TAP_DANCE_DOUBLE(KC_COMM, KC_QUOT),
-  [BACKSPACE] = ACTION_TAP_DANCE_DOUBLE (KC_BSPACE, LCTL(KC_BSPACE)),
-  [TAP_TAB] = ACTION_TAP_DANCE_DOUBLE (KC_TAB, LSFT(KC_TAB)),
-  [CTRL_MINUS] = ACTION_TAP_DANCE_DOUBLE (KC_LCTL, KC_MINS),
-  [CTRL_PLUS] = ACTION_TAP_DANCE_DOUBLE (KC_RCTL, KC_EQL)
-};
-#endif
+#include "custom_keycodes.h"
+#include "timer_utils.h"
 
 #if defined(RGB_MATRIX_ENABLE)
 extern void eeconfig_update_rgb_matrix_default(void);
 #endif
 
 #ifdef TRILAYER_ENABLED
-uint32_t layer_state_set_user(uint32_t state) {
-  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+uint32_t layer_state_set_user(uint32_t state)
+{
```

Comment:
Why?!

Reply:
Because I have a huge monitor

Reply:
lol, but .... eeeew

Also, technically:
https://docs.qmk.fm/#/contributing?id=coding-conventions

Reply:
ya, but mah keymapz

Reply:
lol.  ..... fine. 

But yes, in general, we do want to keep the code style consistent.... as possible.

Though, that's more important for core files, I suppose.